backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt

```diff
@@ -52,4 +61,39 @@ class SpotifyPlaylistsTests {
         // assertEquals(playlist.description, createPlaylistRequestBody.description)
         assertEquals(playlist.owner.id, userId)
     }
+
+    @Test
+    fun getPlaylistDetails() {
+        // given
+        val playlistId = "XD"
+        val mockService = mock(SpotifyPlaylistsService::class.java)
+        val mockPlaylist = SpotifyPlaylist(
+            description = null,
+            externalUrls = ExternalUrls(
+                spotify = "spotify"
+            ),
+            id = "id",
+            images = listOf(Image(
+                url = "url",
+                height = 10,
+                width = 10,
+            )),
+            name = "name",
+            owner = Owner(
+                externalUrls = ExternalUrls(
+                    spotify = "spotify"
+                ),
+                displayName = "name",
+                id = "id"
+            ),
+            public = true
+        )
+
+        //when
```

Comment:
https://github.com/akai-org/fillist/security/code-scanning?query=pr%3A69+tool%3Aktlint+is%3Aopen 
fix ktlint alerts
for most, run gradle ktlintFormat is enougth